src/text/index.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { Fragment } from 'react'
+import cc from 'classcat'
+
+import style from 'text/style'
+
+export enum TextDisplayType {
+  BODY = 'body',
+  BODYSTRONG = 'bodyStrong',
+  BUTTON = 'button',
+  CAPTION = 'caption',
+  DISPLAY1 = 'display1',
+  DISPLAY2 = 'display2',
+  SUBHEADER = 'subheader',
+  SUBHEADERSTRONG = 'subheaderStrong',
+  TITLE = 'title',
+  TITLESTRONG = 'titleStrong',
+}
+
+interface TextProps {
+  readonly className?: Classcat.Class,
+  readonly children: string | number | React.ReactNode,
+  readonly display?: TextDisplayType,
+  readonly div?: boolean,
+}
+
+const Text = ({
+  className,
+  children,
+  display = TextDisplayType.BODY,
+  div = false,
```

Comment:
can it only be a div? No bold, italic etc.?

Reply:
This is handled by "display", which corresponds to the different types of text shown in the design system